tests/models/test_rcf.py

```diff
@@ -37,7 +37,8 @@ def test_untrainable(self) -> None:
 
     def test_biases(self) -> None:
         model = RCF(features=24, bias=10, mode="gaussian")
-        assert torch.all(model.biases == 10)
+        # https://github.com/pytorch/pytorch/issues/116328
+        assert torch.all(model.biases == 10)  # type: ignore[call-overload]
```

Comment:
Seems like a bug in mypy to me, possibly related to https://github.com/pytorch/pytorch/issues/116328. Mypy thinks the output of `model.biases == 10` is a bool, when it should be a bool Tensor.